src/svc.rs

```diff
@@ -38,7 +45,12 @@ impl<L> Builder<L> {
         D: buffer::Deadline<Req>,
         Req: Send + 'static,
     {
-        self.layer(buffer::layer(bound, d)).layer(pending::layer())
+        self.layer(buffer::layer(bound, d)).pending()
+    }
+
+    /// Buffer requests when when the next layer is out of capacity.
```

Comment:
This comment is the same as fn pending. Should this be describing a layer that ensures readiness is driven on the inner service?